md/single-sign-on-with-saml.md

```diff
@@ -226,7 +226,7 @@ Your IdP should declare a Service Provider named `bonita` (or the value of the `
 - the Name ID or a user attribute of the user principal sent back by the IdP should match the username of the user accounts in Bonita BPM and the PrincipalNameMapping policy (and attribute value) in **keycloack-saml.xml** should reflect that
 
 ::: info
-**Note:** If the IdP declares a redirect/target URL, it might orveride the target URL set by the service provider request, and you may always end up on the same page after logging in. In that case, try to remove the redirect URL. Bonita BPM supports redirection to the URL initially requested after logging in on the IdP provided the IdP doesn't force this URL.
+**Note:** If the IdP declares a redirect/target URL, it might orveride the target URL set by the Service Provider request, and you may always end up on the same page after logging in. In that case, try to remove the redirect URL. Bonita BPM supports redirection to the URL initially requested after logging in on the IdP provided the IdP doesn't force this URL.
 :::
 
 ## Troubleshoot
```

Comment:
I don't think that logging out is about trouble shooting.
It's a special use case, isn't it?

Reply:
Actually the section ## Configure logout behaviour is at the same level as Troubleshoot. But we should probably put the Troubleshoot section at the end...